packages/imodel-select/package.json

```diff
@@ -2,34 +2,43 @@
   "name": "@bentley/imodel-select-react",
   "version": "2.0.2",
   "description": "iModel Select React",
-  "license": "MIT",
-  "main": "lib/imodel-select-react.js",
-  "typings": "lib/imodel-select-react",
-  "scripts": {
-    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
-    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
-    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize",
-    "build-prod": " tsc -p tsconfig.json 1>&2 && npm run copy:assets",
-    "clean": "rimraf lib build",
-    "lint": "tslint -p . 1>&2",
-    "test": "",
-    "cover": ""
-  },
   "keywords": [
     "Bentley",
     "iModel Select React",
     "iModel.js"
   ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/imodel-select/packages/imodel-select"

```

Comment:
Repo Url doesn't exist, I believe it should be https://github.com/imodeljs/viewer-components-react/tree/master/packages/imodel-select

Reply:
nice catch!